fire_atlas_editor/lib/utils/select_file.dart

```diff
@@ -1,18 +1,18 @@
 import 'dart:html';
 
 void selectFile(void Function(String) callback) {
-  InputElement uploadInput = FileUploadInputElement();
+  final uploadInput = FileUploadInputElement();
   uploadInput.click();
 
   uploadInput.onChange.listen((e) {
     // read file content as dataURL
     final files = uploadInput.files;
-    if (files.length == 1) {
-      final file = files[0];
+    if (files?.length == 1) {
+      final file = files![0];
       final reader = new FileReader();
 
       reader.onLoadEnd.listen((e) {
-        callback(reader.result);
+        if (reader.result != null) callback(reader.result! as String);
       });
       reader.readAsDataUrl(file);
     }
```

Comment:
you probably wanna show a message here? like Please select exactly one file or something?